content/source/docs/enterprise/api/variables.html.md

```diff
@@ -97,14 +101,14 @@ curl \
 
 ## List Variables
 
-| Method | Path           |
-| :----- | :------------- |
-| GET | /vars |
+`GET /vars`
+
+### Query Parameters
 
-### Parameters
+[These are standard URL query parameters](./index.html.html#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D`.
 
-- `?filter[organization][name]` (`optional`) - Optionally filter the list to an organization given the organization name
-- `?filter[workspace][name]` (`optional`) - Optionally filter the list to a workspace given the workspace name
+- `filter[organization][name]` (optional) - Optionally filter the list to an organization. If omitted, returns data from all organizations the user has permission to view.
+- `filter[workspace][name]` (optional) - Optionally filter the list to a workspace. If omitted, returns data from all workspaces the user has permission to view.
```

Comment:
Are these clarifications true?

Reply:
@davidcelis Ah, thanks. So, you can include zero (shows everything) or both (shows one workspace)? Or put differently, it's just a workspace filter, but we need org because we're filtering workspaces by name instead of ID?

Reply:
@nfagerlund Yup, that's right!